content/200-concepts/100-components/02-prisma-client/030-crud.mdx

```diff
@@ -777,6 +777,70 @@ To resolve this error, you can:
   const transaction = await prisma.$transaction([deletePosts, deleteUser])
   ```
 
+### Delete all records from all tables
+
+Sometimes you want to remove all data from all tables but keep the actual tables. This can be particularly useful in a development environment.
+
```

Comment:
```suggestion
Sometimes you want to remove all data from all tables but keep the actual tables. This can be particularly useful in a development environment. In the following two sections, you can learn how to delete all records from all tables with Prisma Client and with Prisma Migrate.

#### Deleting all records with Prisma Client

```